src/CorpusLoaders.jl

```diff
@@ -12,12 +12,13 @@ export load
 
 export WikiCorpus, SemCor, Senseval3
 
-function __init__()
+file_path = dirname(@__FILE__)
 
-    include("WikiCorpus_DataDeps.jl")
-    include("SemCor_DataDeps.jl")
-    include("SemEval2007Task7_DataDeps.jl")
-    include("Senseval3_DataDeps.jl")
+function __init__()
+    include(joinpath(file_path, "WikiCorpus_DataDeps.jl"))
+    include(joinpath(file_path, "SemCor_DataDeps.jl"))
+    include(joinpath(file_path, "SemEval2007Task7_DataDeps.jl"))
+    include(joinpath(file_path, "Senseval3_DataDeps.jl"))
```

Comment:
This joinpath filepath stuff is not required.
Include is already relative

Reply:
This was throwing an error otherwise. On travis ( I guess appveyor would show similar ) -
```
ERROR: LoadError: InitError: could not open file /home/travis/build/JuliaText/CorpusLoaders.jl/test/WikiCorpus_DataDeps.jl
```

Or on my system of the form - 
```
julia> using CorpusLoaders
[ Info: Recompiling stale cache file /home/ayushk4/...]
ERROR: InitError: could not open file /home/ayushk4/WikiCorpus_DataDeps.jl
```
I deduced that include inside the `__init__ function` takes the path relative to the directory being called from, rather than relative to the location of the module, so I corrected it accordingly. What other alternatives could be taken for this?

Reply:
Oh that is weird, Weird weird weird.
If that is indeed the case 
then you can at least do 
```
 include(joinpath(@__DIR__, "SemCor_DataDeps.jl"))
```

Reply:
Here's the error on [Travis](https://travis-ci.org/JuliaText/CorpusLoaders.jl/jobs/540649432#L560) and [Appveyor](https://ci.appveyor.com/project/oxinabox/corpusloaders-jl-iwgrp/builds/24996907/job/dh05i5smlpax8h5f#L132) without joinpath, if you may be interested.